config/prow/config_knative.yaml

```diff
@@ -1042,15 +1046,19 @@ periodics:
     - continuous: true
       go113: true
       dot-dev: true
+      needs-dind: true
     - nightly: true
       go113: true
       dot-dev: true
+      needs-dind: true
     - dot-release: true
```

Comment:
The `dot-release` is missing `dot-dev: true`, maybe it was missed when it was added in the first time?